definitions/npm/formik_v0.9.x/test_formik.js

```diff
@@ -3,6 +3,9 @@
 import * as React from "react";
 import { Formik, Form, Field, type FieldProps } from "formik";
 
+// $ExpectError
+const foo: number = "bar";
```

Comment:
I know it's/I'm nitpicking here, but can you replace this with a broken version of what's below it? The entire point is to test that incorrect use of `Formik` will throw errors. Something like:

```javascript
<Formik
  // $ExpectError
  initialProperties={{ foo: "hi" }}
/>;
```

Reply:
Not nit picking at all. I must have been feeling really lazy. Updated